lib/l10n/l10n.dart

```diff
@@ -7,10 +7,12 @@
 /// TexturedPolak@github => Polish
 /// yuri-val@github => Ukrainian
 /// energywave@github, ncvescera@github, OpenCode@github => Italian
-/// mdksec@github => Turkish
+/// mikropsoft@github => Turkish
```

Comment:
```suggestion
/// mdksec@github, mikropsoft@github => Turkish
```

You can't remove the previous contributor but you can add your name next to his/her name

Reply:
that's my first username, that's why I changed it @KRTirtho

Reply:
> that's my first username, that's why I changed it @KRTirtho

I am the one who sent the file through the community :)

Reply:
Just because you're reborn doesn't mean your old existence has to be erased /s

Kidding, I'll change it back 😄